src/h5web/toolbar/controls/DomainSlider/DataHistogram.tsx

```diff
@@ -8,11 +9,13 @@ import { adaptedNumTicks } from '../../../vis-packs/core/utils';
 import styles from './DataHistogram.module.css';
 
 interface Props {
-  bins: number[];
   data: number[];
-  size: Size | undefined;
+  bins: number[];
   scaleType: ScaleType;
   sliderDomain: Domain;
+  colorMap?: ColorMap;
```

Comment:
`colorMap` is kept optional as the domain slider could be used to other purposes that setting the domain of a color bar.